### ISSUE TITLE
Extending nanowm

### ISSUE THREAD

**Issue by lighth7015** [ghaerr/microwindows#38]
Hey all, another thread for a new topic. So I'm planning around with the innards of nanowm and have noticed that while trying to theme it similarly to Win16, it mis-calc's the size of the client's borders, causing the client area overflows by 1px.

I'm sure I am doing something, but I'm curious what it would be.

I will post more in a bit, as I'm on my phone right now.

Thanks!

**Comment by lighth7015** [comment]
Okay, so this is a screenshot of my rendering-
![image](https://user-images.githubusercontent.com/671336/78204187-eecb2b80-7466-11ea-9bf6-0274f77a9d71.png)

**Comment by ghaerr** [comment]
Hi @lighth7015, that is very cool!!!

Do you have the corner drags and stuff working?

Can you make the Windows look selectable with a compile-time option? It would be great to add it, providing you're willing! It looks quite nice.

**Comment by lighth7015** [comment]
Oh yeah! I am using an IFDEF classic, anyway why would it be mis-calc'ing the size of the client area, @ghawrr? I also have improvemental to the Windows 9x default theme.

**Comment by lighth7015** [comment]
like it actually looks like NT4. :p

**Comment by lighth7015** [comment]
@ghaerr These are my changes to the Windows 9x style,
![image](https://user-images.githubusercontent.com/671336/78320299-243f4a00-7537-11ea-9b8d-bf21d97b74c5.png)

**Comment by ghaerr** [comment]
Robert, your changes are quite nice!

What more precisely is the client area oversize problem? Are you talking about a single-line border around it, or something else?

**Comment by lighth7015** [comment]
Yes, when resizing the window, I think it uses the sizing data from the old style.
